include/nbl/system/IApplicationFramework.h

```diff
@@ -52,6 +52,8 @@ class NBL_API IApplicationFramework : public core::IReferenceCounted
             return onAppTerminated_impl();
         }
 
+        virtual void onResize(uint32_t w, uint32_t h) {}
```

Comment:
Move this to IGraphicalApplication framework. And store an instance of the IGraphicalApplication in the EventHandler